src/app/classes/entities/creatures/monsters/zombie.js

```diff
@@ -0,0 +1,16 @@
+import { Assets } from '../../../assets/assets';
+import { Monster } from './monster';
+
+export class Zombie extends Monster {
+  constructor(handler, x, y) {
+    super(handler, x, y);
+
+    this.baseAttack = 20;
+
+    this.assets = Assets.getAssets('zombie');
+  }
+
+  getAnimationFrame() {
+    return this.assets.animations['walk_left'].getCurrentFrame();
+  }
+}
```

Comment:
needs newline at end of file 👍

Reply:
interesting, I've never seen that being done before, but you prefer that there's a new line at the end of files?

Reply:
Yea, It's not usually an issue but it can cause some weirdness so it's just good practice:

https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/